### ISSUE TITLE
Missing function in latest version

### ISSUE THREAD

**Issue by 2shediac** [blindsidenetworks/moodle-mod_recordingsbn#17]
- with the version 2016051905 when trying to add to a course get the following error:
Exception - Call to undefined function bigbluebuttonbn_get_moodle_version_major()

**Comment by mudrd8mz** [comment]
We've run into it too. It seems that the commit 82e126cc1deef46a55865a03495b99a30c9559bd removed call to that function.

**Comment by jfederico** [comment]
All the functionality that RecordingsBN has is now included in BigBlueButtonBN. The two plugins were merged in one, which means that users who updated to 2.2 no longer need RecordingsBN. 

BigBlueButtonBN has now three possible types that users can choose when adding/updating the activity. 

- Room and recordings (which is the same as it was before), 
- Room only (which is basically a room with recording functionality turned off)
- Recordings only (that is all the functionality provided by RecordingsBN)

The reason behind is that as the functionality for managing recordings increased we started getting too much duplication. RecordingsBN will be still there for old versions, but no longer updated for the new ones.

If we wanted to keep supporting the RecordingsBN for working with the newest version of BigBlueButtonBN, It is actually not as simple as updating the calls to the functions but also extending the library in BigBlueButtonBN to be used by an external plugin. That is a major update to both plugins.

The new use case is, users can add as many rooms with no recordings as they need for the live sessions and only one instance in mode "Recordings only".

The only downside so far is that for those courses or templates already created, administrators would need to manually remove the instances of RecordingsBN and replace them with a BigBlueButtonBN one in "Recordings only" mode.
